www/docs/customization/nfpm.md

```diff
@@ -362,6 +368,12 @@ nfpms:
         # Templates: allowed.
         key_file: "{{ .Env.GPG_KEY_PATH }}"
 
+      # Set the arch.
+      #
+      # Default: inferred from binary's GOARCH based on the format.
+      # Since: v1.22.
+      arch: arm64
```

Comment:
the configs in the docs are not meant to be correct, only valid... each option should be thought about in its on merit.. if we aim to make it consistent... it would be a loooooooot of work and a lot more docs I think